tasks/sep/base-002-fp-upgrade/SignFromJson.s.sol

```diff
@@ -0,0 +1,366 @@
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "overhead()(uint256)"`
+    uint256 constant gasPriceOracleOverhead = 0;
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "scalar()(bytes32)"`
+    uint256 constant gasPriceOracleScalar = 0x010000000000000000000000000000000000000000000000000a118b0000044d;
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "startBlock()(uint256)"`
+    uint256 constant systemConfigStartBlock = 4370903;
+
+    AddressManager addressManager = AddressManager(0x709c2B8ef4A9feFc629A8a2C1AF424Dc5BD6ad1B);
+
+    address ownerSafe;
+
+    Types.ContractSet proxies;
+
+    /// @notice Sets up the contract
+    function setUp() public {
+        proxies = _getContractSet();
+
+        ownerSafe = vm.envAddress("OWNER_SAFE");
+    }
+
+    function getCodeExceptions() internal pure override returns (address[] memory) {
+        address[] memory shouldHaveCodeExceptions = new address[](4);
+
+        shouldHaveCodeExceptions[0] = systemConfigOwner;
+        shouldHaveCodeExceptions[1] = batchSenderAddress;
+        shouldHaveCodeExceptions[2] = p2pSequencerAddress;
+        shouldHaveCodeExceptions[3] = batchInboxAddress;
+
+        return shouldHaveCodeExceptions;
+    }
+
+    /// @notice Reads the contract addresses from lib/superchain-registry/superchain/extra/addresses/sepolia/base.json
+    function _getContractSet() internal returns (Types.ContractSet memory _proxies) {
+        string memory addressesJson;
+
+        // Read addresses json
+        try vm.readFile(
+            string.concat(vm.projectRoot(), "/lib/superchain-registry/superchain/extra/addresses/sepolia/base.json")
+        ) returns (string memory data) {
+            addressesJson = data;
+        } catch {
+            revert("Failed to read lib/superchain-registry/superchain/extra/addresses/sepolia/base.json");
+        }
+
+        _proxies.L1CrossDomainMessenger = stdJson.readAddress(addressesJson, "$.L1CrossDomainMessengerProxy");
+        _proxies.L1StandardBridge = stdJson.readAddress(addressesJson, "$.L1StandardBridgeProxy");
+        _proxies.OptimismMintableERC20Factory =
+            stdJson.readAddress(addressesJson, "$.OptimismMintableERC20FactoryProxy");
+        _proxies.OptimismPortal = stdJson.readAddress(addressesJson, "$.OptimismPortalProxy");
+        _proxies.OptimismPortal2 = stdJson.readAddress(addressesJson, "$.OptimismPortalProxy");
+        _proxies.SystemConfig = stdJson.readAddress(addressesJson, "$.SystemConfigProxy");
+        _proxies.L1ERC721Bridge = stdJson.readAddress(addressesJson, "$.L1ERC721BridgeProxy");
+
+        // Read superchain.yaml
+        string[] memory inputs = new string[](4);
+        inputs[0] = "yq";
+        inputs[1] = "-o";
+        inputs[2] = "json";
+        inputs[3] = "lib/superchain-registry/superchain/configs/sepolia/superchain.yaml";
```

Comment:
similar to the above, this is now a `toml` file.

Reply:
Same here, updated